chart/templates/secrets.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.extraSecrets }}
```

Comment:
`{{- if .Values.extraSecrets }}` is not needed here and can be removed.
If the list is empty - no op.